src/test/java/runner/TestRunner.java

```diff
@@ -3,30 +3,42 @@
 import java.io.File;
 
 import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import com.cucumber.listener.Reporter;
+import com.vimalselvam.cucumber.listener.Reporter;
 import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
+import helpers.ConfigReader;
+import helpers.FileOperations;
 
 @RunWith(Cucumber.class)
 @CucumberOptions(
         features = "src/test/resources/features",
         glue={"stepsdefinition"},
         //tags = {""},
-        plugin = {"com.cucumber.listener.ExtentCucumberFormatter:target/cucumber-reports/report.html"},
+        plugin = {"com.vimalselvam.cucumber.listener.ExtentCucumberFormatter:target/cucumber-reports/report.html"},
```

Comment:
What adds more this new plugin?

Reply:
Is the same plugin than before. I upgrade the version of vimaselsvam to the lastone and, in this new version, it changes the package name.